### DISCUSSION TITLE
0.6.0

### DISCUSSION THREAD

**Discussion by sakethpathike** [LinkoraApp/Linkora#3] [General]
## Updates
- **Shelf** can now be minimized or maximized using a new floating action button above the "Add" button on the Home Screen. Linkora will remember your choice.
- Linkora will remember the last panel you used from the left navigation rail and will open it automatically the next time you use the app.
- The limit of 5 panels is gone; you can add as many panels as you want, and the Shelf UI has been updated.
- Items in the shelf navigation rail are now called "Panels". The feature is still called **Shelf**.
- The bottom sheet for managing the Shelf has been replaced with new screens for adding and managing folders in panels.
- The Link UI has been improved.
- The link menu now shows an image at the top if available; otherwise, it shows the regular menu.
- Added "Refresh Title and Image" to update the title and image of a link.
- The full-screen dialog for adding new links lets you add links directly to subfolders.
- The "Refresh all links titles and images" setting now works in the background, and you can cancel it if needed.
- [vxTwitter](https://github.com/dylanpdx/BetterTwitFix) is now included to get Twitter metadata when saving links from Twitter.
- Auto-Update and manual checks for the latest versions now pull data from official Linkora releases.

This update also includes various minor improvements and UI changes.

---

Technically, this is the biggest update Linkora has had since its initial release. The codebase has been mostly rewritten and is now much better. This rewrite is one reason for the delay in this release. The codebase will be improved further if needed.

**Full Changelog**: https://github.com/sakethpathike/Linkora/compare/release-v0.5.0...release-v0.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/sakethpathike/Linkora/releases/tag/release-v0.6.0'>0.6.0</a>.</em>

**Comment by IAAJL** [comment]
Why does the app say root folders data migrated on first launch?

**Reply by sakethpathike** [reply]
The database schema was changed, and if anyone has used it before, it just notifies that _you're up to date_ with the latest database changes. I'll remove that toast message with the next release.
